src/builders/enzymes.jl

```diff
@@ -14,6 +14,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+$(TYPEDEF)
+
+A simple struct storing information about the isozyme composition, including
+subunit stoichiometry and turnover numbers. Use with
+[`enzyme_constrained_flux_balance_analysis`](@ref).
+
+# Fields
+$(TYPEDFIELDS)
+"""
+Base.@kwdef mutable struct Isozyme
+    gene_product_stoichiometry::Dict{String,Float64}
+    kcat_forward::Maybe{Float64} = nothing
+    kcat_reverse::Maybe{Float64} = nothing
+end
```

Comment:
this is a different isozyme than in frontend?

isozymes only belong to frontend.

Reply:
(in short, struct Isozyme should never be touched in builders, it's a frontend-only thing that must be eradicated as soon as possible)

Reply:
same struct, but I would like to keep it, because a useful workflow is to "add enzyme constraints" to an already existing model. But this requires that isozymes already exists somewhere